app-libs/stf/src/stf_sgx.rs

```diff
@@ -182,6 +194,19 @@ impl Stf {
 		})
 	}
 
+	/// Creates valid enclave account with a balance that is above the existential deposit.
+	/// !! Requires a root to be set.
```

Comment:
Can you clarify? Because currently, and it seems to pass CI, no root is set. Do you mean `root_bypass_filter`?

Reply:
In CI we always compile with feature `test`, so we have Alice as root. Only when production is enabled, the `test` feature won't be present (and a root has to be set specifically)

Reply:
I see, thanks for clarifying